src/backend/access/transam/xlog.c

```diff
@@ -748,7 +751,15 @@ typedef struct XLogCtlData
 	 * XLOG_FPW_CHANGE record that instructs full_page_writes is disabled.
 	 */
 	XLogRecPtr	lastFpwDisableRecPtr;
-	XLogRecPtr  lastWrittenPageLSN;
+
+	/*
+	 * Cache of last written page LSN.
+	 * We store this value for up to LAST_WRITTEN_CACHE_SIZE relations + maximum for all other relations.
+	 */
+	XLogRecPtr	lastWrittenPageLsn;
```

Comment:
```suggestion
	XLogRecPtr	lastWrittenPageNonCacheLsn;
```

I know there is a comment described this above, but it's still better to make the variable self-explained. At first, I thought the non-cache relations will be default to use the latest written LSN based on the variable name.